src/plugins/autocompletion_providers/Git.ts

```diff
@@ -50,9 +51,65 @@ const configOptions = combine([
     longAndShortFlag("edit"),
 ]);
 
+const fetchOptions = combine([
+    "quiet",
+    "verbose",
+    "append",
+    "upload-pack",
+    "force",
+    "keep",
+    "depth=",
+    "tags",
+    "no-tags",
+    "all",
+    "prune",
+    "dry-run",
+    "recurse-submodules=",
+].map(option => longFlag(option)));
+
+const commonMergeOptions = combine([
+    "no-commit",
+    "no-stat",
+    "log",
+    "no-log",
+    "squash",
+    "strategy",
+    "commit",
+    "stat",
+    "no-squash",
+    "ff",
+    "no-ff",
+    "ff-only",
+    "edit",
+    "no-edit",
+    "verify-signatures",
+    "no-verify-signatures",
+    "gpg-sign",
+    "quiet",
+    "verbose",
+    "progress",
+    "no-progress",
+].map(option => longFlag(option)));
+
+const remotes = async(context: AutocompletionContext): Promise<Suggestion[]> => {
+    if (Git.isGitDirectory(context.environment.pwd)) {
+        const names = await Git.remotes(context.environment.pwd);
+        return names.map(name => new Suggestion().withValue(name).withStyle(styles.branch));
+    }
+
+    return [];
+};
+
 const configVariables = unique(async(context: AutocompletionContext): Promise<Suggestion[]> => {
     const variables = await Git.configVariables(context.environment.pwd);
-    return variables.map((variable) => new Suggestion().withValue(variable).withStyle(styles.option));
+
+    return variables.map((variable) => {
```

Comment:
There is shorter syntax for it: `map(variable => new Suggestion().withThings())`